File: Assignment 4 copy/Student.java
public class Student extends Person{
	

	private int year;
	
	
	Student(String name, String address, String phone, int year) {
		
		super(name,address,phone);

		this.year = year;
		//this keyword refers to current object in a method or constructor and most common use of the keyboard is to avoid confusion between class attributes and parameters of the same name	
			
		
	}
	
	

void setGraduationYear(int year) {//void indicates method doesn't have return value
	this.year = year;
	
	
}

public int getGraduationYear() {
	
	return year;
}

public String toString() {//returns string representation of the object
	return super.toString() + "\nYear: " + year;	
	
}




}

File: Assignment 3 backup copy/Student.py
from Person import Person
class Student(Person):

    def __init__(self,newName="none", address="none", phone="999-999-9999", year=9999):
        super().__init__(newName,address,phone)
        self.year = year
 
    def getGraduationYear(self):
        return (self.year)
    
    def setGraduationYear(self,year):
        self.year = year

    def __str__(self) -> str:
        return super().__str__() + "Year:" + str(self.year)


        



    


        